docs/launch-with/bdcat-launch-with.rst

```diff
@@ -1,7 +1,7 @@
 BioData Catalyst
 ================
 
-Dockstore integrates with the `NHLBI BioData Catalyst <https://biodatacatalyst.nhlbi.nih.gov/>`__ platform. You can
+Dockstore integrates with the `NHLBI BioData Catalyst <https://dockstore.org/organizations/bdcatalyst>`__ platform. You can
```

Comment:
This is a nice idea, linking to their dockstore page rather than an external source